documentation/src/main/asciidoc/topics/con_evict_vs_expire.adoc

```diff
@@ -5,8 +5,8 @@ they remove old, unused entries. Although eviction and expiration are similar,
 they have some important differences that you should take into account when
 planning your configuration.
 
-* [*] Eviction prevents {brandname} from exceeding the maximum size of the data
-container. {brandname} performs eviction when you add entries to the cache.
+* [*] Eviction lets {brandname} control the size of the data container.
+{brandname} performs eviction when you add entries to the cache.
```

Comment:
```suggestion
* [*] Eviction lets {brandname} control the size of the data container by evicting entries when the container becomes larger than a configured threshold.
```
Wdyt?

Reply:
@wburns sounds good but I'd tweak it a little to say "by removing entries". it's kind of like we're using the word eviction to explain what eviction means.